config via configmaps (#63)
logzio/logzio-k8s: README.md

### To deploy logzio-k8s

#### 1. Create a monitoring namespace

> This is the namespace that the Daemonset will be deployed under.

Reviewer: ```suggestion
Your Daemonset will be deployed under this namespace.
```
